lib/usd/utils/util.cpp

```diff
@@ -138,31 +138,13 @@ defPrimSpecLayer(const UsdPrim& prim)
     return defLayer;
 }
 
-
 SdfPrimSpecHandle 
 getPrimSpecAtEditTarget(const UsdPrim& prim)
 {
     auto stage = prim.GetStage();
     return stage->GetEditTarget().GetPrimSpecForScenePath(prim.GetPath());
 }
 
-bool
-isInternalReference(const SdfPrimSpecHandle& primSpec)
-{
-    bool isInternalRef{false};
-
-    for (const SdfReference& ref : primSpec->GetReferenceList().GetAddedOrExplicitItems()) {
-        // GetAssetPath returns the asset path to the root layer of the referenced layer
-        // this will be empty in the case of an internal reference.
-        if (ref.GetAssetPath().empty()) {
-            isInternalRef = true;
-            break;
-        }
-    }
-
-    return isInternalRef;
-}
```

Comment:
This peace of code didn't make much sense and it is not used anymore, therefore removing it.